presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestResourceGroupsDao.java

```diff
@@ -117,14 +117,12 @@ private static void testSelectorUpdate(H2ResourceGroupsDao dao, Map<Long, Select
                 2,
                 Optional.of(Pattern.compile("ping.*")),
                 Optional.of(Pattern.compile("ping_source")));
-        map.remove(2);
```

Comment:
Related: i was using intellin inspections to double-check i found every place to update, and this was showing up. It's separate commit, why not keep it in the PR? (I won't open another PR with just this, waste of time).

How can i improve the commit message?